docassemble/MassAccess/data/questions/massaccess.yml

```diff
@@ -67,4 +68,76 @@ code: |
 ---
 code: |
   github_user = "suffolklitlab"  
-  
\ No newline at end of file
+---
+template: al_terms_of_use
+subject: ""
+content: |
+  <h2 class="h5"> Terms of Use and Privacy Policy</h2>
+
+  This website is free to use. Suffolk University Law School's Legal Innovation and Technology Lab ("LIT Lab"), a nonprofit, hosts the site.
+
+  And we include information from:
+
+  * Massachusetts Law Reform Institute
+  * Massachusetts Legal Assistance Corporation
+  * Greater Boston Legal Services
+  * Northeast Legal Aid
+
+  <h3 class="h5">You need to know</h3>
+
+  * This website does not constitute or provide legal advice. You should consult a lawyer for legal advice. If you need assistance finding a Lawyer you can use [https://masslrf.org](https://masslrf.org)to find one. 
+  * We use our best efforts to prepare the forms on this site and to keep them up to date. Law changes or court or local variations may make any forms generated by this site unenforceable.
+  * This website is offered completely without warranty, including any implied warranties. The LIT Lab disclaims all liability for your use of information on this website or your reliance on the forms created by a guided interview.
+  * You agree to indemnify and hold harmless the LIT Lab and the Law School for all claims arising out of your use of these guided interviews or the information that they give you.
+  * The LIT Lab does not offer individual technical support for the interviews provided on this website.
+  * Information you enter may not be protected by attorney-client privilege or attorney work product doctrine.
+  * Do not use this system to conduct unauthorized practice of law. If you do, you may face civil or criminal liability.
+
+  <h3 class="h5">To use this site</h3>
+
+  * You need to be over 13 to use this site.
+  * This site is for **anyone** 13 and over. For example: you can be the person who needs the help, a lawyer, or a social worker.
+  * You may not sell the information or the contents of this site. But you can charge for your own time.
+  * Please keep your use fair. Do not use this site in a way that is illegal or that makes it harder for other people to use.
+
+  <h3 class="h5">Information we and others collect</h3>
+
+  * We collect the information that you type to help you complete your forms. We delete all information 180 days after you last update it. You can also delete information immediately.
```

Comment:
Not urgent, but curious: do we store things for 180 days elsewhere? On production, `interview delete days` isn't set so I think it defaults to 90 days. We can change it to 180 if we want, I just didn't know where else we might be keeping things